README.md

```diff
@@ -1,3 +1,69 @@
 # Mollie PHP Coding Standards
 
 Contains PHP coding standards like rules for PHP-CS-Fixer that serves for purpose of standardization.
+
+## Usage
+
+Place a file named `.php_cs.dist` that has following content in your project's root directory.
+
+```php
+use PhpCsFixer\Config;
+use PhpCsFixer\Finder;
+
+$finder = Finder::create()
+    ->name('.php_cs.dist')       // Fix this file as well
+    ->in(__DIR__);
+
+return Config::create()
+    ->setFinder($finder)
+    ->setRiskyAllowed(true)
+    ->setRules(Rules::getForPhp71());
```

Comment:
I miss a bit of explanation that this line is the relevant part of this package, and the fact that we also have 7.2 and 7.3 versions if you're at that version.